taxonomy-editor-frontend/src/pages/root-nodes/index.tsx

```diff
@@ -84,7 +85,7 @@ const RootNodes = ({
     );
   }
 
-  if (isPending || !nodes) {
+  if (isPending || !nodes || nodes.length === 0) {
```

Comment:
I think there is an issue with this implementation: the message will show every time we open a project until the nodes are fetched, even if the project is already parsed and open, which is not what indicates the displayed message.

This message should be displayed only if the project status is LOADING